docs/source/tracking/autolog.rst

```diff
@@ -176,8 +176,8 @@ Autologging captures the following information:
 
 .. _autolog-keras:
 
-Keras
------
+Keras/TensorFlow
+----------------
 Call the generic autolog function or :py:func:`mlflow.tensorflow.autolog` before your training code to enable automatic logging of metrics and parameters. As an example, try running the `Keras/Tensorflow example <https://github.com/mlflow/mlflow/blob/master/examples/keras/train.py>`_.
```

Comment:
did not create a separate section for TensorFlow cuz it's going to be a pure copy-paste of this section. decided to merge them together. let me know what everyone thinks